pkg/compiler/codegen.go

```diff
@@ -1629,15 +1629,14 @@ func (c *codegen) resolveFuncDecls(f *ast.File, pkg *types.Package) {
 	}
 }
 
-func (c *codegen) writeJumps(b []byte) error {
+func (c *codegen) writeJumps(b []byte) ([]byte, error) {
 	ctx := vm.NewContext(b)
+	var indices []int
```

Comment:
I guess these are no longer indices, but rather offsets.

Reply:
These are indices in the `b` slice. "offsets" is misleading, because we already have offsets (instruction parameter).